Mathlib/Geometry/Manifold/SmoothManifoldWithCorners.lean

```diff
@@ -133,15 +133,18 @@ scoped[Manifold] notation "∞" => (⊤ : ℕ∞)
 
 /-! ### Models with corners. -/
 
+section ModelWithCorners
+
+variable (𝕜 : Type*) [NontriviallyNormedField 𝕜]
+  (E : Type*) [NormedAddCommGroup E] [NormedSpace 𝕜 E]
+  (H : Type*) [TopologicalSpace H]
 
 /-- A structure containing informations on the way a space `H` embeds in a
 model vector space `E` over the field `𝕜`. This is all what is needed to
 define a smooth manifold with model space `H`, and model vector space `E`.
 -/
 @[ext] -- porting note: was nolint has_nonempty_instance
-structure ModelWithCorners (𝕜 : Type*) [NontriviallyNormedField 𝕜] (E : Type*)
-    [NormedAddCommGroup E] [NormedSpace 𝕜 E] (H : Type*) [TopologicalSpace H] extends
-    LocalEquiv H E where
+structure ModelWithCorners extends LocalEquiv H E where
```

Comment:
I'm skeptical this is a readability improvement.

I don't mind `variables` for long blocks of theorems, but at the initial definition they are definitely unhelpful.

Reply:
I've restored the variable declarations for the `def`s. I'm trying to calibrate my intuition based on your comment. Hopefully the rest is appropriate.